wagl/psf.py

```diff
@@ -248,6 +247,33 @@ def format_tp5(json_data: dict) -> Tuple[dict, dict]:
     return data, kwargs
 
 
+def create_dataset(
```

Comment:
Possibly a bit of overkill, as you could get away with just:

```python
ds = group.create_dataset(band_name, data=data, compression='lzf')
attach_attribute(ds, attrs)
```

Or for compression filter consistency use `compression.config`.

This is fine for the time being, and if we need to look at it again later we can.

Reply:
Overkill in relation to defining a function for outputting the kernel.